docs/application/native/guides/personal/account.md

```diff
@@ -314,7 +315,9 @@ To add your application on the Add account screen of the device:
 
    - `<icon section="account">application icon name</icon>`
 
-     Icon displayed on the Add account screen.The icon size must be 72 x 72 px and the icon file must be in the `{app-root}\shared\res\` directory.
+     Icon displayed on the Add account screen.
+
+     The icon size must be 72 x 72 px and the icon file must be in the `{app-root}\shared\res\` directory.
 
    - `<icon section="account-small">application small icon name</icon>`
 
```

Comment:
line 324: The separated lines are merged as one line. (The small icon file must)
line 864: The separated lines are merged as one line, Code tag(data type) was not applied. (Table: Account properties)
line 897: The separated lines are merged as one line, Code tag(data type) was not applied. (Table: Account provider properties)

Reply:
line 324: fixed.
the others: Why don't you see my previous comment?